string name           # Levha adı
float64 confidence    # Confidence değeri
float64 distance      # Orijine uzaklık
